### PR TITLE
Feature#53/syle product form

### PR DESCRIPTION
**Pull request by germanruzca**
# Description
 What?: 
 - I've added the styles and functionality to the Form of create a new post (Product)
 - Create new JS file to add the logic when pictures are uploaded to the browser.
 - Why?: To have a nice user experience with the design and the view of the form
  
  ## Screenshots 
image
image

fogh

### CODE REVIEW

File: app/javascript/addItem.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	let size = imgs.length;
+	for (let key = 0; key <= size; key++) {
+		parent.removeChild(parent.lastChild);
+	}
+}
+
+window.deleteImage = function(position) {
+	images.splice(position, 1);
+	showImages(images);
+}
```
Comment (YilKanda) [inline app/javascript/addItem.js:34]: Check your indentation (Space: 2)

**Review by Javi1499 — APPROVED** [review]
LGTM

File: app/assets/stylesheets/components/_new_post_form.scss [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+  }
+  .new_post__view {
+    padding: 20px;
+    background-color: $white-light;
+    border: 1px solid black;
+    width: 800px;
+    margin: 40px;
+    box-shadow: 2px 2px 5px;
+  }
+
```
Comment (nallely64) [inline app/assets/stylesheets/components/_new_post_form.scss:164]: I suggest to order this block

**Review by nallely64 — CHANGES_REQUESTED** [review]

**Review by nallely64 — CHANGES_REQUESTED** [review]
I suggest to order the .new_post__view styles

File: app/assets/stylesheets/components/_new_post_form.scss (hunk 2 above)
Comment (germanruzca) [reply app/assets/stylesheets/components/_new_post_form.scss:164]: Done!

File: app/assets/stylesheets/components/_new_post_form.scss [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+  background-color: $white-light;
+  display: flex;
+  flex-direction: column;
+}
+
+.new_post__view {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
```
Comment (EliasUR) [inline app/assets/stylesheets/components/_new_post_form.scss:12]: You can use sass sintax here using &
Example:
```suggestion
.new_post {
  // content
  &__view {
    // content
  }
}
```

Comment (germanruzca) [reply app/assets/stylesheets/components/_new_post_form.scss]: Done, thank you

**Review by Javi1499 — APPROVED** [review]
LGTM

**Review by nallely64 — APPROVED** [review]
LGTM!

**Pull request merged**